### ISSUE TITLE
is it required to install caffe before ssdcaffe .?

### ISSUE THREAD

**Issue by dlounis** [weiliu89/caffe#806]
Hi,

is it required to install caffe before ssdcaffe .?

Regards.

**Comment by novate** [comment]
Not at all, this branch has contained a modified caffe already. More than that, if you've already installed original caffe, you should remove it.
